zuul.d/trigger-jobs.yaml

```diff
@@ -0,0 +1,13 @@
+- job:
+    name: architecture-downstream-va-hci-trigger-job
+    parent: downstream-va-hci-trigger-job
+    description: |
+      This job will trigger a VA HCI job downstream.
+      We can invoke this job by adding following comment
+      `\trigger github-experimental` in the github pull request comment.
+      These jobs runs in github-experimental-trigger pipeline.
+    files:
+      - ^lib
```

Comment:
we should probably start having one file per DT/VA, instead of the main "defaults.yml" nested in the automation tree. That would allow to trigger the job upon automation file modification as well.

@fultonj @abays IIRC the first proposal for the automation description file was, indeed, to get one file per VA/DT. IMHO, having everything in one single file was a mistake - we can probably correct that without the need to break all of the existing jobs, by ensuring the filename matches the scenario name.

We'd then "just" need a simple change in the framework to consume the file, and voilà.

Any thoughts?

Otherwise, the file listed here looks OK, with the current automation layout.

Reply:
Some VA/DTs already have dedicated files, so I think we can move the ones to a dedicated file for each VA/DT without any issue.

Reply:
I have no objections to having a separate file per VA/DT.

Reply:
If we get a PR which proposes separate file per VA/DT, then I'd be happy to help merge it.

Reply:
As a way forward can do this?

- ci-framework patch and architecture patch to split default into multiple files
- then we merge this patch

Reply:
Above approach looks good to me. I will work on splitting the defaults changes and then merge this patch.